api/cpp/include/slint_platform.h

```diff
@@ -427,6 +427,18 @@ inline void update_timers_and_animations()
     cbindgen_private::slint_platform_update_timers_and_animations();
 }
 
+/// Returns the duration until the next timer or std::chrono::milliseconds::max() if
+/// there are no pending timers
+inline std::chrono::milliseconds duration_until_next_timer_update()
```

Comment:
I think return an std::optional makes for a nicer API on the caller side when they want to check for this condition.